### ISSUE TITLE
betteruptime_status_page does not support history parameter

### ISSUE THREAD

**Issue by dleborgne** [BetterStackHQ/terraform-provider-better-uptime#21]
Status page API and corresponding web interface both support a `history` parameter used to set the number of days to display on the status page. Available values in the web interface are "90 days", "180 days" and "365 days"

betteruptime_status_page does not support such parameter. The following example with result in a Terraform unsupported argument error: An argument named "history" is not expected here.

```
resource "betteruptime_status_page" "this" {
  company_name             = "MyCompany"
  company_url              = "https://www.mycompany.com/"
  subdomain                = "status-mycompany"
  timezone                 = "Paris"
  history                  = 180
}
```

After removing the `history` parameter, the status pages created with the betteruptime_status_page Terraform will have a default 90 days status history.

**Comment by adikus** [comment]
@dleborgne Thanks for bringing this to our attention. I've opened a PR that adds the history attribute.
I'll ping you again once we get it merged and released.

**Comment by adikus** [comment]
@dleborgne This should be now released as version 0.3.8
